iot_config/src/region_map.rs

```diff
@@ -65,10 +65,14 @@ pub async fn build_region_tree(
 
     let mut regions = sqlx::query_as::<_, HexRegion>("select * from regions").fetch(db);
 
-    while let Some(region_row) = regions.try_next().await? {
-        let region = Region::from_i32(region_row.region)
-            .ok_or(RegionMapError::UnsupportedRegion(region_row.region))?;
-        let mut h3_idx_decoder = Decoder::new(&region_row.indexes[..])?;
+    while let Some(HexRegion {
+        region,
+        indexes: Some(indexes),
+        ..
+    }) = regions.try_next().await?
```

Comment:
This will stop processing the first row it encounters that doesn't have indexes.
Is there any chance a row without indexes will be sandwiched between others that do?

Reply:
Darn, you’re right. Please hold

Reply:
fixed [df46ecb](https://github.com/helium/oracles/pull/348/commits/df46ecbd7189f68169b6e84f79ea577815f397de)